modules/mcc/include/opencv2/mcc/ccm.hpp

```diff
@@ -0,0 +1,522 @@
+    */
+    CV_WRAP ColorCorrectionModel(const Mat& src, Mat colors, COLOR_SPACE ref_cs);
+
+    /** @brief Color Correction Model
+    @param src detected colors of ColorChecker patches;\n
+                the color type is RGB not BGR, and the color values are in [0, 1];
+    @param colors the reference color values,the color values are in [0, 1].
+    @param ref_cs the corresponding color space
+                NOTICE: For the list of color spaces supported, see the notes above;\n
+                If the color type is some RGB, the format is RGB not BGR;
+    @param colored mask of colored color
+    */
+    CV_WRAP ColorCorrectionModel(const Mat& src, Mat colors, COLOR_SPACE ref_cs, Mat colored);
+
+    /** @brief set ColorSpace
+
+    @note It should be some RGB color space;
+    Supported list of color cards:
+    - @ref COLOR_SPACE_sRGB,
+    - @ref COLOR_SPACE_AdobeRGB,
+    - @ref COLOR_SPACE_WideGamutRGB,
+    - @ref COLOR_SPACE_ProPhotoRGB,
+    - @ref COLOR_SPACE_DCI_P3_RGB,
+    - @ref COLOR_SPACE_AppleRGB,
+    - @ref COLOR_SPACE_REC_709_RGB,
+    - @ref COLOR_SPACE_REC_2020_RGB,
+    - @param cs the absolute color space that detected colors convert to;\n
+              default: @ref COLOR_SPACE_sRGB
+    */
+    CV_WRAP void setColorSpace(COLOR_SPACE cs);
+
+    /** @brief set ccm_type
+    @param ccm_type the shape of color correction matrix(CCM);\n
+                    default: @ref CCM_3x3
+    */
+    CV_WRAP void setCCM_TYPE(CCM_TYPE ccm_type);
+
+    /** @brief set Distance
+    @param distance the type of color distance;\n
+                    default: @ref DISTANCE_CIE2000
+    */
+    CV_WRAP void setDistance(DISTANCE_TYPE distance);
+
+    /** @brief set Linear
+
+    Supported list:
+    @param linear_type the method of linearization;\n
+                       default: @ref LINEARIZATION_GAMMA
+    */
+    CV_WRAP void setLinear(LINEAR_TYPE linear_type);
+
+    /** @brief set Gamma
+
+    @note only valid when linear is set to "gamma";\n
+
+    @param gamma the gamma value of gamma correction;
+                 default: 2.2;\n
+    */
+    CV_WRAP void setLinearGamma(double gamma);
+
+    /** @brief set degree
+    @param deg the degree of linearization polynomial;\n
+        NOTICE: only valid when linear is set to
+        @ref LINEARIZATION_COLORPOLYFIT
+        @ref LINEARIZATION_GRAYPOLYFIT
+        @ref LINEARIZATION_COLORLOGPOLYFIT
+        @ref LINEARIZATION_GRAYLOGPOLYFIT
+               default: 3
+    */
+    CV_WRAP void setLinearDegree(int deg);
+
+    /** @brief set SaturatedThreshold
+    @param lower the lower threshold to determine saturation;\n
+            default: 0;
+    @param upper the upper threshold to determine saturation;
+        NOTICE: it is a tuple of [lower, upper];
+                The colors in the closed interval [lower, upper] are reserved to participate
+                in the calculation of the loss function and initialization parameters\n
```

Comment:
Looks like notice is related for both parameters. It make sense to put this description as method body.

Reply:
Done!